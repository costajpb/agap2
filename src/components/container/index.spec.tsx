import { render } from '@testing-library/react'
import Container from '.'

describe('components/container', () => {
    it('should render', () => {
        const { container } = render(<Container>Dummy text</Container>)
        expect(container).toMatchSnapshot()
    })
})
