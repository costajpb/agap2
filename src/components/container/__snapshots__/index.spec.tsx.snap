// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/container should render 1`] = `
<div>
  <div
    class="sc-bXCLTC bpxPoR"
  >
    Dummy text
  </div>
</div>
`;
